src/compas/datastructures/mesh/core/_mutablemapping.py

```diff
@@ -31,7 +31,7 @@ class Mapping(object):
     """
 
     def get(self, key, default=None):
-        'D.get(k[,d]) -> D[k] if k in D, else d.  d defaults to None.'
+        'D.get(k[,d]) => D[k] if k in D, else d.  d defaults to None.'
```

Comment:
ha sorry, that is not what i meant. the docstrings should be enclosed in `"""` not `'` or `'''` or `"`

Reply:
`->` and `=>` are both fine by me :)

Reply:
ahh! true, fixed at https://github.com/compas-dev/compas/pull/563/commits/a352f6b758db9bb4d70d4ad58a3d7bf0cdf5da33